base/iostream.jl

```diff
@@ -321,13 +321,18 @@ function peek(s::IOStream)
     ccall(:ios_peekc, Cint, (Ptr{Void},), s)
 end
 
+
+function _seekback(pos::Integer, s::IOStream)
```

Comment:
I changed both for consistency. Most of the other `ios` functions do use the io argument first too, so I think that's alright.